### PR TITLE
Blocksparse rowmajor

### PR DESCRIPTION
**Pull request by mganahl**
switch from column to row major ordering for unique charges

### CODE REVIEW

**Comment by mganahl** [comment]
It looks more than it actually is, mostly tiny changes + some better formatting

File: tensornetwork/block_sparse/charge.py [hunk 1]
```diff
@@ -223,22 +221,28 @@ def __add__(self, other: "BaseCharge") -> "BaseCharge":
     Returns:
       BaseCharge: The result of fusing `self` with `other`.
     """
+    if len(self.charge_labels) == 0 or len(other.charge_labels) == 0:
+      obj = self.__new__(type(self))
+      obj.__init__(
+          np.empty((0, self.num_symmetries), dtype=self.dtype),
```
Comment (Thenerdstation) [inline tensornetwork/block_sparse/charge.py:227]: What's the point of having an array with a dimension size of 0?

File: tensornetwork/block_sparse/charge.py [hunk 2]
```diff
@@ -691,8 +696,12 @@ def intersect(A: np.ndarray,
     ndarray: the indices of the first occurrences of the common values in B.
       Only provided if return_indices is True.
   """
-  #see https://stackoverflow.com/questions/8317022/get-intersecting-rows-across-two-2d-numpy-arrays
+  # see
+  # https://stackoverflow.com/questions/8317022/
+  # get-intersecting-rows-across-two-2d-numpy-arrays
```
Comment (Thenerdstation) [inline tensornetwork/block_sparse/charge.py:701]: Keep the URL on one line.

File: tensornetwork/block_sparse/charge.py (hunk 1 above)
Comment (mganahl) [reply tensornetwork/block_sparse/charge.py:227]: For consistency the class needs to be initialized

**Comment by mganahl** [comment]
somethings still wrong with this, so don't merge it yet

**Pull request merged**